### PR TITLE
Alternative Languages

### PR DESCRIPTION
**Pull request by quark-zju**
Add an "Alternative Languages" option, which is a list of languages, allowing the player to:

- Search cards using these languages
- View texts of these languages in card tooltip

This is quite useful for players active in HS communities of different languages.

Screenshot:

altlang

### CODE REVIEW

**Comment by Epix37** [comment]
Very nice, I like it. 

Minor issue I found is that if you have no additional languages selected, the tooltip has some unnecessary whitespace at the bottom:
spacing

**Comment by quark-zju** [comment]
I was aware of the padding issue but TBH I have no idea how these XAML gird layout works and how to solve the issue. I use Linux for development most of the time. Some little help will be greatly appreciated.

File: Hearthstone Deck Tracker/Controls/CardToolTipControl.xaml [hunk 1]
```diff
@@ -38,7 +39,11 @@
             <TextBlock Name="CardTextBlock" Text="{Binding Text}" Grid.Row="1" FontSize="14"
                        Foreground="{DynamicResource TextBrush}" VerticalAlignment="Top" FontWeight="Medium"
                        HorizontalAlignment="Center" TextWrapping="Wrap" Margin="10,0,10,0" TextAlignment="Center" />
-            <Grid Grid.Row="2" Background="{DynamicResource AccentColorBrush}">
+            <TextBlock Name="AlternativeLanguageText" Text="{Binding AlternativeLanguageText}" Grid.Row="2" FontSize="12"
+                       Foreground="#FF7F7F7F" VerticalAlignment="Top"
+                       Visibility="{Binding ShowAlternativeLanguageTextInTooltip}"
```
Comment (quark-zju) [inline Hearthstone Deck Tracker/Controls/CardToolTipControl.xaml]: I tried to hide the TextBlock showing alternative texts by setting Visibility property here. However it does not work as expected. Do you have some idea?

**Comment by quark-zju** [comment]
Nevermind, I think I got it :)

**Comment by Epix37** [comment]
Looks good now.


Another issue I found:
* Alternative language active
* Config.Instance.UseFullTextSearch 
* Search for something in edit

results in: ` NullReferenceException at MainWindow_NewDeck.cs:Line 87`

Card.Text is null if the text is empty.

**Comment by quark-zju** [comment]
Nice catch! I think it's okay now.

**Comment by Epix37** [comment]
Yup, seems to be all working correctly now! :+1:

**Pull request merged**